Source/OxyPlot/Axes/Axis.cs

```diff
@@ -817,6 +818,8 @@ public virtual void Measure(IRenderContext rc)
         {
             this.GetTickValues(out var majorLabelValues, out _, out _);
 
+            majorLabelValues = majorLabelValues.Where(d => d > this.ActualMinimum && d < this.ActualMaximum).ToList();
```

Comment:
The test passes if we write >= and <= here... but that is not a complete solution, because the renderer also uses an epsilon when it checks if it should render a particular label: 
https://github.com/oxyplot/oxyplot/blob/9ffa2a89fdaac66f4cbe7f5cd1a527d02a6c5f87/Source/OxyPlot/Axes/Rendering/HorizontalAndVerticalAxisRenderer.cs#L379

So basically we have no safe way of knowing here whether a label is actually rendered or not...

On the other hand, the worst that could happen is that we reserve slightly too much space for a label that isn't rendered. So maybe just use >= and <= and be done with it?

Reply:
So I think there is no way to do this properly as long as we can't be sure which labels are actually rendered and which are not.
Why is it even the business of the renderer to decide which labels are rendered? Maybe we should move this responsibility out of the renderer altogether... I try something and see how it goes.

Reply:
I have many times lamented how inextensible axes are: I'd be very happy to help in any way I can with refitting them (testing, design, implementation) so that this sort of thing makes more sense. When I've tried in the past (and I've tried a few times), I have always given up and gone back to doing what I was meant to be doing before deciding to re-write the axes.